SAML-OIDC/WebApp_SAML_OIDC/README.md

```diff
@@ -0,0 +1,337 @@
+---
+page_type: sample
+languages:
+  - csharp
+products:
+  - dotnet
+  - aspnet
+  - azure-active-directory  
+name: Signing in users with SAML and OpenIDConnect in the same web application
+urlFragment: ms-identity-dotnet-adfs-to-aad
+---
+
+# Signing in users with SAML and OpenIDConnect in the same web application
+
+ 1. [Overview](#overview)
+ 1. [Scenario](#scenario)
+ 1. [Contents](#contents)
+ 1. [Prerequisites](#prerequisites)
+ 1. [Setup](#setup)
+ 1. [Registration](#registration)
+ 1. [Running the sample](#running-the-sample)
+ 1. [Explore the sample](#explore-the-sample)
+ 1. [About the code](#about-the-code)
+ 1. [Deployment](#deployment)
+ 1. [More information](#more-information)
+ 1. [Community Help and Support](#community-help-and-support)
+ 1. [Contributing](#contributing)
+
+![Build badge](https://identitydivision.visualstudio.com/_apis/public/build/definitions/a7934fdd-dcde-4492-a406-7fad6ac00e17/<BuildNumber>/badge)
+
+## Overview
```

Comment:
>Overview [](http://example.com/codeflow?start=3&length=8)

Add a section 

## Disclaimer This sample does not intend to lay down the pattern of foundation of how this complex situation should be approaced or done. Thi s is instead provided as a good starting point for developers to start with and experiment further till it meeds their needs.